node-server/server.js

```diff
@@ -107,10 +109,20 @@ const setupCacheHeaders = app => {
   );
 };
 
+// NOTE: This is where we replace the HTML with the up-to-date runtime environment.
+//
+// This requires the `replace-in-file` NPM package.
```

Comment:
C'est discutable mais je pense qu'on a pas besoin de cette ligne. La fonction du package est appelé, on sait donc qu'on en dépend

Reply:
Yup, these comments are temporary while this is still a draft 🙂